tests/test_goldpoems.py

```diff
@@ -0,0 +1,26 @@
+import json
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from tests.test_utils import TESTS_DIR
+
+from averell.readers.goldpoems import get_features
+from averell.readers.goldpoems import parse_xml
+
+
+@pytest.fixture
+def goldpoems():
+    return json.loads(Path("tests/fixtures/goldpoems.json").read_text())
+
+
+def test_parse_xml(goldpoems):
+    poem = parse_xml("tests/fixtures/input_goldpoems.xml")
```

Comment:
Try to avoid `tests/fixtures/disco.json` and use a proper Path() instead for win/unix compatibility

Reply:
Done